### PR TITLE
[RELAY][TENSORFLOW] Support tf.reverse()

### PR DESCRIPTION
**Pull request by lixiaoquan**
Support tf.reverse()

### CODE REVIEW

**Comment by yongwww** [comment]
@lixiaoquan I have added the reverse (and Tile, Round, etc.) support in pr #2960

File: tests/python/frontend/tensorflow/test_forward.py [hunk 1]
```diff
@@ -1268,6 +1268,21 @@ def test_forward_rel_ops():
     _test_forward_rel_op([t1, t2], math_ops.equal)
     _test_forward_rel_op([t1, t2], math_ops.not_equal)
 
+#######################################################################
+# Reverse
+# ---------
```
Comment (srkreddy1238) [inline tests/python/frontend/tensorflow/test_forward.py]: Underline up to text length pls.

File: tests/python/frontend/tensorflow/test_forward.py [hunk 2]
```diff
@@ -1268,6 +1268,21 @@ def test_forward_rel_ops():
...
+def _test_forward_reverse(shape, axis):
+    inp_array = np.random.uniform(-5, 5, size=shape).astype(np.float32)
+    with tf.Graph().as_default():
+        in1 = tf.placeholder(shape=inp_array.shape, dtype=inp_array.dtype)
+        out = tf.reverse(in1, axis=[axis])
+        compare_tf_with_tvm(inp_array, 'Placeholder:0', out.name)
+
+def test_forward_reverse():
+    _test_forward_reverse((2, 3, 10, 10), 0)
```
Comment (srkreddy1238) [inline tests/python/frontend/tensorflow/test_forward.py]: Can you add a test case with negative axis too ?

**Comment by srkreddy1238** [comment]
I think  #2960 implements reverse operator.

**Comment by lixiaoquan** [comment]
#2960 is better, I'll close this one.
